src/generateRandomNumber.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generateRandomNumber = () => {
+  const randomDigits = [];
+
+  for (let i = 0; i < 4; i++) {
+    const randomNumber = Math.floor(Math.random() * 9000) + 1000;
```

Comment:
in this way you already generate 4 digits random number that looks like 1***, so you dont need a loop and shafl it